code/datums/gamemode/factions/bloodcult/bloodcult_items.dm

```diff
@@ -1,16 +1,350 @@
+
+	if (iscultist(user) || isobserver(user))
+		if (attuned_rune)
+			to_chat(user, "<span class='info'>This one was attuned to a <b>[initial(instance.name)]</b> rune.</span>")
+		else
+			to_chat(user, "<span class='info'>This one was imbued with a <b>[initial(instance.name)]</b> rune.</span>")
+	else
+		to_chat(user, "<span class='info'>This one was some arcane drawings on it. You cannot read them.</span>")
+
+/obj/item/weapon/talisman/attack_self(var/mob/living/user)
+	if (iscultist(user))
+		trigger(user)
+
+/obj/item/weapon/talisman/proc/trigger(var/mob/user)
+	if (!user) return
+
+	if (!spell_type)
+		if (src != user.get_active_hand())
+			user.put_in_hands(src)//removing from a tome
+		return
+
+	if (attuned_rune)
+		attuned_rune.trigger(user)
+	else
+		new spell_type(user, src)
+
+	if (istype(loc,/obj/item/weapon/tome))
+		var/obj/item/weapon/tome/T = loc
+		T.talismans.Remove(src)
+	qdel(src)
+
+/obj/item/weapon/talisman/proc/imbue(var/mob/user, var/obj/effect/rune/R)
+	if (!user || !R) return
+
+	var/datum/rune_spell/spell = get_rune_spell(user,null,"examine",R.word1, R.word2, R.word3)
+	if(initial(spell.talisman_absorb) == RUNE_CANNOT)
+		user.drop_item(src)
+		src.forceMove(get_turf(R))
+		R.attack_hand(user)
+	else
+		if (attuned_rune)
+			to_chat(user, "<span class='warning'>This [src] is already linked to a rune.</span>")
+			return
+		if (attuned_rune)
+			to_chat(user, "<span class='warning'>This talisman is already imbued with the power of a rune.</span>")
+			return
+
+		if (!spell)
+			to_chat(user, "<span class='warning'>There is no power in those runes. The talisman isn't reacting to it.</span>")
+			return
+
+		if (initial(spell.Act_restriction) > 1000)//TODO: SET TO CURRENT CULT FACTION ACT
+			to_chat(user, "<span class='danger'>The veil is still too thick for a talisman to draw power from this rune.</span>")
+			return
+
+		//blood markings
+		overlays += image(icon,"talisman-[R.word1.icon_state]a")
+		overlays += image(icon,"talisman-[R.word2.icon_state]a")
+		overlays += image(icon,"talisman-[R.word3.icon_state]a")
+		//black markings
+		overlays += image(icon,"talisman-[R.word1.icon_state]")
+		overlays += image(icon,"talisman-[R.word2.icon_state]")
+		overlays += image(icon,"talisman-[R.word3.icon_state]")
+
+		spell_type = spell
+		switch(initial(spell.talisman_absorb))
+			if (RUNE_CAN_ATTUNE)
+				user.playsound_local(src, 'sound/effects/talisman_attune.ogg', 50, 0, 0, 0, 0)
+				to_chat(user, "<span class='notice'>The talisman can now remotely trigger the [initial(spell.name)] rune.</span>")
+				attuned_rune = R
+			if (RUNE_CAN_IMBUE)
+				user.playsound_local(src, 'sound/effects/talisman_imbue.ogg', 50, 0, 0, 0, 0)
+				to_chat(user, "<span class='notice'>The talisman absorbs the power of the [initial(spell.name)] rune.</span>")
+				qdel(R)
+			if (RUNE_CANNOT)
+				message_admins("Error! Some bloke managed to imbue a Conjure Talisman rune. That shouldn't be possible!")
```

Comment:
Give some info - who is the `usr` and the `src` for starters. Maybe even a `stack_trace` for debug purposes.